apps/docs/docs/reference/ui/molecules/Card.mdx

```diff
@@ -80,4 +76,26 @@ All Card-related components support all attributes also supported by the `<div>`
 
 `data-store-card-content`
 
-`data-store-card-actions`
\ No newline at end of file
+`data-store-card-actions`
+
+## Best practices
+
+### Do's
+
+- Use attractive images to grab shoppers' attention.
+- Keep the card content short.
+- Keep a standard visual style. Otherwise, your design may look off-putting.
+- Use transparent-backgrounded images.
+- Use animation and visual hints to improve the user experience.
+- Use the most appropriate card size for each device type.
+- Make the entire card clickable to create larger touch zones.
+
+### Don'ts
+
+- Don't use the `Card` component for product cards. Instead, use the [`ProductCard` component](/reference/ui/molecules/ProductCard).
+- Avoid cluttering cards with too many buttons and content. 
+- Avoid inline links.
+
+## Accessibility
+
+- Text elements and buttons must have sufficient color contrast against the background.
```

Comment:
We can also add:
- For images containing text, the `alt` description must includes the image's text.